app/models/user.rb

```diff
@@ -249,8 +250,9 @@ def set_remember_fields
   end
 
   def calculate_score
-    score = (articles.where(featured: true).size * 100) + comments.sum(:score)
-    update_column(:score, score)
+    user_reaction_points = Reaction.user_vomits.where(reactable_id: id).sum(:points)
```

Comment:
Just to make sure we're on the same page, these are "negative" points, i.e. the count against the user, right? What about the positive reactions, do we completely disregard them?

Reply:
Also, does summing positive reactions mean that non-spammy authors will have scores now -- since `user.calculate_score` was unused before?

Reply:
Correct me if I'm wrong and if I misunderstand either of the questions:

- Yes, these are all negative points, and to my understanding a _user_ cannot receive positive points _at all_. So you mean positive reactions to their posted comments/articles?
- Yes, non-spammy authors will now have scores. I left another comment below on what the implications of that are.

The _only_ thing score technically does now is allow a flagged user to be `noindex` unless the reaction is dismissed by admins. A small part of a larger overall spam mitigation approach. So with that I dumbed it down to only look at badge counts, but very open to expanding on this and writing the definitive bible for how we should think about calculating scores in general. I ultimately thought of that as outside the scope of this PR, but that this was all logically safe to act this way for the time being.

Don't mistake my acting with intentionality with _being right_ about this, or that I haven't made a mistake somewhere. Definitely don't want to ship a thing which accidentally `noindex`'s all user profiles. 😅

Reply:
> Yes, these are all negative points, and to my understanding a user cannot receive positive points at all. 

I guess this is what I wanted to explicitly see written out, thanks!